website/static/js/oauthAddonNodeConfig.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Module that controls the addon node settings. Includes Knockout view-model
+ * for syncing data, and HGrid-folderpicker for selecting a folder.
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
Is knockout punches really necessary? I only see two uses of `{{ }}` syntax in the template; try to avoid using an entire extra library when regular knockout syntax would suffice.